examples/air_temperature_gullingen_skisenter-parent_meta.json

```diff
@@ -0,0 +1,51 @@
+{
+    "type": "Feature",
+    "geometry": {
+        "type": "Point",
+        "coordinates": [
+            59.4128,
+            6.4697
+        ]
+    },
+    "properties": {
+        "title": "Air temperature observations from weather station Gullingen , ROGALAND, Norge, (WIGOS ID: 0-578-0-46220, station ID: 46220).",
+        "data_id": "8a0744b7-f0dc-4929-8700-f1285e76df36",
```

Comment:
From MQTT def: "A universally unique identifier of the message using the UUID standard".

I suggest to create a new uuid, as the one provided refers to the parent dataset. I see the docs require uuid, but it may be better to combine it with a namespace. Let's make a new issue for the latter.